pkg/expapi/types.go

```diff
@@ -264,3 +264,98 @@ type DeploymentStatus struct {
 	// Total number of new ready pods with the desired template spec.
 	UpdatedReplicas int `json:"updatedReplicas,omitempty"`
 }
+
+// Job represents the configuration of a single job.
+type Job struct {
+	api.TypeMeta   `json:",inline"`
+	api.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec is a structure defining the expected behavior of a job.
+	Spec JobSpec `json:"spec,omitempty" description:"definition of the expected behavior of a job"`
+
+	// Status is a structure describing current status of a job.
+	Status JobStatus `json:"status,omitempty" description:"description of a current job status"`
+}
+
+// JobList is a collection of jobs.
+type JobList struct {
+	api.TypeMeta `json:",inline"`
+	api.ListMeta `json:"metadata,omitempty"`
+
+	Items []Job `json:"items" description:"list of jobs"`
+}
+
+// JobSpec describes how the job execution will look like.
+type JobSpec struct {
+
+	// Parallelism specifies the maximum desired number of pods the job should
+	// run at any given time. The actual number of pods running in steady state will
+	// be less than this number when ((.spec.completions - .status.successful) < .spec.parallelism),
+	// i.e. when the work left to do is less than max parallelism.
+	Parallelism *int `json:"parallelism,omitempty" description:"maximum number of pods the job should run"`
```

Comment:
From my own tests it looks like it's not verified, so per your suggestion I'm removing those from both places. And thank you for reading those from the comment, this will save quite a lot of duplication.